Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.10-alpine
+LABEL maintainer="leon.kushnir15@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    my_user

```

Comment:
Consider using a non-root user for running the application to enhance security. You've already created a user `my_user`, which is a good practice. Ensure that the application runs with this user.